drupal8-mysql/docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+version: '3.7'
+services:
+  db:
```

Comment:
Can you add a healthcheck please? Some of the other mysql samples should have one.